### ISSUE TITLE
Manual trigger: Enable directus to post data on EAS using custom service

### ISSUE THREAD

**Issue by ChialiT** [Enaleia/Directus#14]
Relate to https://github.com/Enaleia/eas-lib/issues/4

When admin complete the data and ready to send to EAS,
we will provide a button which is manual trigger in directus so admin can send data to EAS.

this task will involve 2 phases,
1: on OP testnet (we will test this flow on staging, once work, move to phase 2)
2: on OP mainnet

**Comment by ChialiT** [comment]
json mapping and sequence flow at: https://www.figma.com/board/CJqXAUt1A7HAgLsM7fr9mU/Enaleia?node-id=4733-13822&t=jpb7BGoQbe5iJuv4-4

@kinizumi

**Comment by kinizumi** [comment]
was able to get 200 response
```
{
  "status": 200,
  "statusText": "OK",
  "headers": {
    "content-length": "85",
    "content-type": "application/json; charset=utf-8",
    "date": "Tue, 25 Feb 2025 14:44:49 GMT",
    "etag": "W/"55-84U2P9msDW9HUGuSymXB7FLAQus"",
    "ngrok-agent-ips": "65.181.9.81",
    "x-powered-by": "Express",
    "connection": "close"
  },
  "data": {
    "message": "UID: 0xc799a0559371cc5ef1a39efd08a3cc5aa7cf5b41a88d1f11c5613dfaf865d8db"
  }
}
```
with payload
```
{
  "method": "POST",
  "url": "https://7c82-103-235-95-206.ngrok-free.app/attest",
  "body": {
    "userID": "d9993791-edd9-475e-9e6f-187fbade3efd",
    "portOrCompanyName": "Skyplast",
    "portOrCompanyCoordinates": [
      "38.0321, 23.3677"
    ],
    "actionType": "Fishing for litter",
    "actionDate": "2025-02-24T22:38:10.921Z",
    "actionCoordinates": [
      "22.52243204846765,-22.56167428990292"
    ],
    "collectorName": "Abbas Shehata Abdo ",
    "incomingMaterials": [
      "Mixed Plastic"
    ],
    "incomingWeightsKg": [],
    "incomingCodes": [
      "AQ16"
    ],
    "outgoingMaterials": [],
    "outgoingWeightsKg": [],
    "outgoingCodes": [],
    "productName": "test",
    "batchQuantity": "test",
    "weightPerItemKg": "test"
  }
}
```
https://optimism-sepolia.easscan.org/attestation/view/0xc799a0559371cc5ef1a39efd08a3cc5aa7cf5b41a88d1f11c5613dfaf865d8db
however the 4 fields are mandatory can't leave them empty "productName, batchQuantity, weightPerItemKg, collectorName"

**Comment by unidwell** [comment]
> was able to get 200 response

Yeeeey 🚀

> however the 4 fields are mandatory can't leave them empty "productName,
batchQuantity, weightPerItemKg, collectorName"

For EAS payload they can be empty strings or you get an error when doing
that?

**Comment by kinizumi** [comment]
https://hq.enaleia-hub.com/admin/settings/flows/45366970-7861-4a47-9c46-a043183a3bbe
was able to save uid to the target event, tested

**Comment by kinizumi** [comment]
added a recursive flow to handle bulk submit
https://hq.enaleia-hub.com/admin/settings/flows/3a970946-e708-4657-a2f1-46c8a3f1350c

**Comment by Thatsocool** [comment]
@kinizumi i don't think we need this last bit to bulk submit, normally for a flow to apply to multiple records, a selection in list view can be done.

<img width="871" alt="Image" src="https://github.com/user-attachments/assets/78db9eb3-34d2-4e70-b360-70729496d8cc" />

**Comment by ChialiT** [comment]
There is still limitation of sending too many item to post at same time, but using 25 items per batch posting, it will not cause any error
